lms/static/js/student_account/views/FormView.js

```diff
@@ -185,32 +176,32 @@
 
                 saveError: function(error) {
                     this.errors = ['<li>' + error.responseText + '</li>'];
-                    this.setErrors();
+                    this.renderErrors(this.defaultFormErrorsTitle, this.errors);
                     this.toggleDisableButton(false);
                 },
 
-                setErrors: function() {
-                    var $msg = this.$errors.find('.message-copy'),
-                        html = [],
-                        errors = this.errors,
-                        i,
-                        len = errors.length;
-
-                    for (i = 0; i < len; i++) {
-                        html.push(errors[i]);
-                    }
+                renderErrors: function(title, errorMessages) {
+                    this.renderFormFeedback(this.formErrorsTpl, {
+                        title: title,
+                        messagesHtml: HtmlUtils.HTML(errorMessages.join(''))
+                    });
+                },
```

Comment:
This is just meant to be a convenience since we the majority of our calls to renderFormFeedback are to render error messages. Happy to remove it and just have everything call renderFormFeedback if you'd prefer, but those calls would be a bit more verbose.